Firestore/Example/Tests/Core/FSTEventManagerTests.m

```diff
@@ -139,11 +139,11 @@ - (void)testWillForwardOnlineStateChanges {
   FSTQueryListener *fakeListener = OCMClassMock([FSTQueryListener class]);
   NSMutableArray *events = [NSMutableArray array];
   OCMStub([fakeListener query]).andReturn(query);
-  OCMStub([fakeListener clientDidChangeOnlineState:FSTOnlineStateUnknown])
+  OCMStub([fakeListener applyOnlineStateChange:FSTOnlineStateUnknown])
```

Comment:
This doesn't seem to follow Objective-C's naming convention as it does not use a suffix that describes the argument. I'm going to suggest `applyChangeForOnlineState`, but I'm certain that there are better options.

Reply:
Renamed to applyChangedOnlineState: per offline discussion.